pages/index.js

```diff
@@ -12,21 +12,29 @@ export default function HomePage() {
   }
 
   return (
-    <StyledList>
-      {isLoading ? (
-        <li>
-          <h2>Loading...</h2>
-        </li>
-      ) : (
-        items.map((item) => (
-          <li key={item.itemId}>
-            <StyledLink href={`/items/${item.itemId}`}>
-              <Item title={item.title} initialStatus={item.initiallyLost} />
-            </StyledLink>
+    <>
+      <AddItemLink href={"/create"}>
+        <StyledLinkTitle>Add item</StyledLinkTitle>
+      </AddItemLink>
+      <StyledList>
+        {isLoading ? (
+          <li>
+            <h2>Loading...</h2>
           </li>
-        ))
-      )}
-    </StyledList>
+        ) : (
+          items
```

Comment:
this is a fine solution with method chaining, however, you could also choose an approach where most of the logic is not handled within the jsx.
You could for instance have a variable called `reversedItems` that stores a reversed copy of your items and within your jsx you only map over the `reversedItems`.

Reply:
It's temporary. I'll add a map next week, so I don't need reverse() function anymore.